src/base64-inl.h

```diff
@@ -182,6 +211,96 @@ inline size_t base64_encode(const char* src,
   return dlen;
 }
 
+
+#if (defined(__x86_64) || defined(__x86_64__)) && \
+    (defined(__linux) || defined(__linux__))
```

Comment:
I don't see why this operation wouldn't apply outside of Linux? Should this be a check for gcc/clang instead?

Reply:
The original code is for Linux GCC and I do not have a windows to test for it. I would try to remove the line 216 and add gcc/clang check and let the bot to test it. Thanks!